### ISSUE TITLE
To stupid for Node

### ISSUE THREAD

**Issue by Y2-J** [mtadin/steelseries-wireless-battery-tray#32]
What I have to do? Can you upload a video for stupid guys like me? :D

**Comment by mtadin** [comment]
You can download the installer from here:
https://github.com/mtadin/steelseries-wireless-battery-tray/releases

Depending on which headset you have, try either the newerHeadsetsBeta or just the beta version

**Comment by Y2-J** [comment]
Yeah but it wont work. And then I read about that thing with Node in your github…I have to admit I am new at github…Mit freundlichen Grüßen Jörn PohlAm 01.05.24 um 11:06 schrieb Marin Tadin
                
            
            
                
                    Von: "Marin Tadin" ***@***.***>Datum: 1. Mai 2024An: "mtadin/steelseries-wireless-battery-tray" ***@***.***>Cc: "Y2-J" ***@***.***>,"Author" ***@***.***>Betreff: Re: [mtadin/steelseries-wireless-battery-tray] To stupid for Node (Issue #32)
                
                
You can download the installer from here:
https://github.com/mtadin/steelseries-wireless-battery-tray/releases
Depending on which headset you have, try either the newerHeadsetsBeta or just the beta version

**Comment by mtadin** [comment]
https://github.com/mtadin/steelseries-wireless-battery-tray/releases/tag/v3.0.0
